proto/p4/v1/p4runtime.proto

```diff
@@ -284,6 +284,23 @@ message Action {
 
 message ActionProfileActionSet {
   repeated ActionProfileAction action_profile_actions = 1;
+  // Support per-group hash modes and resource usage modes.
+  // Added in v1.5.0.
+  enum ActionSelectionMode {
+    DEFAULT_MODE_DETERMINED_BY_ACTION_SELECTOR = 0;
+    HASH = 1;
```

Comment:
Would be great to add some short comments about what these mean. Tbh, probably the same thing you wrote as bullet points in the spec is fine.